dpnp/fft/dpnp_iface_fft.py

```diff
@@ -86,23 +86,24 @@ def get_validated_norm(norm):
     raise ValueError("Unknown norm value.")
 
 
-def fft(x1, n=None, axis=-1, norm=None):
+def fft(x, n=None, axis=-1, norm=None):
     """
     Compute the one-dimensional discrete Fourier Transform.
 
+    For full documentation refer to :obj:`numpy.fft.fft`.
+
     Limitations
     -----------
-    Parameter ``norm`` is unsupported.
-    Parameter ``axis`` is supported with its default value.
-    Parameter ``x1`` supports ``dpnp.int32``, ``dpnp.int64``, ``dpnp.float32``, ``dpnp.float64``,
-    ``dpnp.complex64`` and ``dpnp.complex128`` datatypes only.
-
-    For full documentation refer to :obj:`numpy.fft.fft`.
+    Parameter `x` is supported either as :class:`dpnp.ndarray`.
+    Parameter `axis` is supported with its default value.
+    Only `dpnp.float64`, `dpnp.float32`, `dpnp.int64`, `dpnp.int32`,
+    `dpnp.complex128`, `dpnp.complex64` data types are supported.
+    Otherwise the function will be executed sequentially on CPU.
```

Comment:
I guess there will an exception rather than fallback on numpy in case of boolean type of input array.
I think we need to clear state that.

Reply:
Probably it would be better to add a type check before calling backend function and to raise an exception for boolean type rather than to have RuntimeError exception.